src/site/apt/version-rules.apt.vm

```diff
@@ -82,11 +88,82 @@ Version number rules
 
   Note: the <<<groupId>>> attribute in the <<<rule>>> elements has a lazy <<<.*>>> at the end, such that
   <<<com.mycompany>>> will match <<<com.mycompany>>>, <<<com.mycompany.foo>>>, <<<com.mycompany.foo.bar>>>, etc.
-  
+
+** Using the <<<ruleSet>>> element in the POM
+
+  As an alternative to providing a separate <<<rules.xml>>> file, starting with version <<<2.13.0>>> it is possible
+  to provide the <<<ruleSet>>> element directly in the POM. The structure is somewhat simpler to
+  the <<<rules.xml>>> file:
+
+---
+<project>
+  ...
+  <build>
+    ...
+    <plugins>
+      ...
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>versions-maven-plugin</artifactId>
+        <version>${pluginVersion}</version>
+        <configuration>
+          <ruleSet>
```

Comment:
```
            <comparisonMethod></..>
```

Reply:
it's there at the end of the <ruleSet/> element